configure

```diff
@@ -317,7 +317,7 @@ desc=sqlite3_expanded_sql
 style=DEFINES_EVERYTHING|EXECUTE|MAY_NOT_COMPILE
 link=$SQLITE3_LDLIBS
 code=
-#include <sqlite3.h>
+#include <sqlite3x.h>
```

Comment:
@rustyrussell: What package provides `<sqlite3x.h>`? Is C-Lightning planning to switch to a fork of SQLite? The SQLite package on my system only provides `<sqlite3.h>` and `<sqlite3ext.h>`. Was the latter renamed at some point upstream? I don't find this header included in any source files, just in these modified `configure` tests.

Reply:
Oops, this is my testing hack!!